services/openshift/scripts/openshift

```diff
@@ -58,7 +58,7 @@ start_openshift() {
         -v ${ORIGIN_DIR}/openshift.local.volumes:${ORIGIN_DIR}/openshift.local.volumes:z \
         -v ${ORIGIN_DIR}/openshift.local.config:${ORIGIN_DIR}/openshift.local.config:z \
         -v ${ORIGIN_DIR}/openshift.local.etcd:${ORIGIN_DIR}/openshift.local.etcd:z \
-        openshift start "${@:2}"
+        $IMAGE start "${@:2}"
```

Comment:
We are also using `openshift` tag to https://github.com/bexelbie/adb-utils/blob/bexelbie_pass_image_info/services/openshift/scripts/openshift#L117 which also need to be updated with this PR.